src/acceptance-test/java/net/consensys/orion/acceptance/send/receive/privacyGroup/PrivacyGroupAcceptanceTest.java

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
Is this an AcceptanceTest file - or base utilities for tests to build up from? I can't see an @Test annotation.

Reply:
The latter - it'll be more widely used as more methods relating to adding & removing privacy groups are added. These are currently implemented in a private branch and PRs for them will be opened in due course.